phase3/Kconfig

```diff
@@ -40,4 +40,10 @@ config phase3.weave_net
 	  Should we deploy weave-net? You likely only want this if using kubeadm for phase2.
 endif
 
+config phase3.cni
```

Comment:
I think this needs to be moved into the above `if phase3.run_addons` context.

And maybe a second if block: `if not phase3.weave_net`?

Reply:
Is there any reason for the second conditional? I'm going to mark `phase.weave_net` as deprecated.